### PR TITLE
Update Issue Template [issue #1266]

### PR DESCRIPTION
**Pull request by Chasmiccoder**
**Description**
Updated Issues templates (Bug Report, Feature Request, Others)

**Related issues or PRs**
Fixes issue #1266

**Changelog**
Update the issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
-A clear and concise description of what you expected to happen.
+
+**Expected Behavior**
+<!-- A clear and concise description of what you expected to happen -->
+
 
 **Your Environment**
-Please use `qutip.about()` to get the information about your environment and paste it here.
+<!-- Please use `qutip.about()` to get the information about your environment and paste it here. -->
+Output of `qutip.about()` -
```
Comment (BoxiLi) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]: @Chasmiccoder Thank you for the PR. LGTM!

Could you also add the following to the "your environment" section? Sometimes people also include the line `===========` of the output, which will be wrongly interpreted as headers of markdown.

Output of `qutip.about()` -
```
QuTiP Version:      ***
Numpy Version:      ***
Scipy Version:      ***
Cython Version:     ***
Matplotlib Version: ***
Python Version:     ***
Number of CPUs:     ***
BLAS Info:          ***
OPENMP Installed:   ***
INTEL MKL Ext:      ***
Platform Info:      ***
```

This should make it clearer that these are the only information needed.

**Comment by jakelishman** [comment]
You might find the new GitHub issues forms useful - I recently swapped Qiskit's templates to those.  See Qiskit/qiskit-terra#7258 for example.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Chasmiccoder) [reply .github/ISSUE_TEMPLATE/bug_report.md:38]: Whoa, totally missed that.
Thanks!

**Comment by Chasmiccoder** [comment]
@jakelishman 
This is awesome! I'll try to implement it

Thanks for the suggestion

**Comment by Chasmiccoder** [comment]
Hello @BoxiLi , @jakelishman .

I implemented the issue forms for Bug Reports, Feature Requests and Other Issues.
Please check them out!

You can test them if you want over here: [test issue forms](https://github.com/Chasmiccoder/Algos-for-Amigos/issues/new/choose)

**Review by BoxiLi — APPROVED** [review]

**Comment by hodgestar** [comment]
This looks great. Thank you!

**Comment by Chasmiccoder** [comment]
Thanks everyone for the help!

**Pull request merged**